net-libs/gwenhywfar/gwenhywfar-5.10.2.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Multi-platform helper library for security applications and libraries"
+DESCRIPTION="OS abstraction functions for various projects"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/gwenhywfar/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2b0b9dd49b95f025f4e0c0346ba88e20b893407c444fccf6403a4da349954f04"
+
+ARCHITECTURES="all"
+
+libVersion="$portVersion"
+
+PROVIDES="
+	gwenhywfar = $portVersion
+	cmd:gct_tool = $portVersion
+	cmd:gsa = $portVersion
+	cmd:gwbuild = $portVersion
+	cmd:gwenhywfar_config = $portVersion
+	cmd:mklistdoc = $portVersion
+	cmd:typemaker = $portVersion
+	cmd:typemaker2 = $portVersion
+	cmd:xmlmerge = $portVersion
+	lib:libgwenhywfar = $libVersion
+	lib:libgwengui_cpp = $libVersion
+	lib:libgwengui_gtk3 = $libVersion
+	"
+
+REQUIRES="
+	haiku
+	lib:libatk_1.0
+	lib:libcairo
+	lib:libcairo_gobject
+	lib:libgcrypt
+	lib:libgdk_pixbuf_2.0
+	lib:libglib_2.0
+   	lib:libgnutls
+	lib:libgpg_error
+	lib:libgtk_3
+	lib:libharfbuzz
+	lib:libiconv
+	lib:libintl
+	lib:libpango_1.0
+	lib:libpangocairo_1.0
+    "
+
+PROVIDES_devel="
+	gwenhywfar_devel = $portVersion
+	devel:libgwenhywfar = $libVersion
+	devel:libgwengui_cpp = $libVersion
+	devel:libgwengui_gtk3 = $libVersion
+	"
+
+REQUIRES_devel="
+	gwenhywfar  == $portVersion base
+	devel:libgcrypt
+	devel:libgnutls
+	devel:libgtk_3
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libgcrypt
+	devel:libgnutls
+	devel:libgtk_3
+    "
+
+BUILD_PREREQUIRES="
+	cmd:autoreconf
+	cmd:gettext
+	cmd:gcc
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	autoreconf -if
+	runConfigure ./configure --with-guis=gtk3 LDFLAGS="-lnetwork"
```

Comment:
When adding 32 bit support, this needs some changes to correct the installation path of the binaries:

```suggestion
	runConfigure --omit-dirs "binDir" ./configure --bindir=$prefix/bin --with-guis=gtk3 LDFLAGS="-lnetwork"
```